README.md

```diff
@@ -2,6 +2,27 @@
 
 `eslint-plugin-relay` is a plugin for [ESLint](http://eslint.org/) to catch common problems in code using [Relay](https://facebook.github.io/relay/) early.
 
+## Install
+`npm i --save-dev eslint-plugin-relay`
+
+## How To Use
+1. Add `"relay"` to your eslint `plugins` section.
+2. Add `"relay/graphql-syntax": "error"` to your eslint `rules` section.
```

Comment:
1. There are multiple rules exposed by the package - https://github.com/relayjs/eslint-plugin-relay/blob/master/eslint-plugin-relay.js#L13L16

2. Also, each rule can be used with different error levels with eslint (and may potentially have their own configuration options) - https://eslint.org/docs/user-guide/configuring#configuring-rules

The line doesn't capture that